src/components/settings/SettingsLMSTab/LMSCard.jsx

```diff
@@ -3,11 +3,15 @@ import PropTypes from 'prop-types';
 import { Card, Image, Stack } from '@edx/paragon';
 import { channelMapping } from '../../../utils';
 
-const LMSCard = ({ LMSType, onClick }) => (
+const LMSCard = ({ LMSType, onClick, disabled }) => (
   <Card
-    isClickable
-    className="pb-4"
-    onClick={() => onClick(LMSType)}
+    isClickable={!disabled}
+    className={`pb-4 ${disabled ? 'lms-card-disabled-opacity' : ''}`}
```

Comment:
for opacity, is it required to have a custom class? in general it's best to use existing util classes if possible, for example, do any of these bootstrap utils work? https://getbootstrap.com/docs/5.1/utilities/opacity/